src/client/src/commons/form/profile/components/profileLayers/profileLayers.js

```diff
@@ -68,6 +74,65 @@ const ProfileLayers = props => {
     },
   );
 
+  const deleteLithologicalDescriptionMutation = useMutation(
+    async id => {
+      const result = await deleteLithologicalDescription(id);
+      return result;
+    },
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries({ queryKey: ["lithoDesc"] });
```

Comment:
REQ: Da der Query-Key nun doch schon an ein paar Orten vorkommt, wäre es angebracht den String in eine Konstante auszulagern und diesen zu benutzen. Dito für den Facies-Key.